text/0000-stack-buildpacks.md

```diff
@@ -0,0 +1,668 @@
+| `<stack-layers>`    | `CNB_STACK_LAYERS_DIR`      | `/stack-layers`           | Path to stack layers directory from extend phase
+
+
+
+#### `rebaser`
+Usage:
+```
+/cnb/lifecycle/rebaser \
+  [-cache-image <cache-image>] \
+  [-plan <plan>] \
+  ...
+```
+
+##### Inputs
+| Input               | Environment Variable  | Default Value          | Description
+|---------------------|-----------------------|------------------------|---------------------------------------
+| ...                 |                       |                        |
+| `<cache-image>`     | `CNB_CACHE_IMAGE`     |                        | Reference to a cache image in an OCI image registry
+| `<plan>`       | `CNB_PLAN_PATH`       | `./plan.toml`     | Path to resolved build plan (see [`plan.toml`](#plantoml-toml))
+| `<stack-platform>`   | `CNB_STACK_PLATFORM_DIR`    | `/stack-platform`       | Path to stack-specific platform directory
+| `<stack-group>`     | `CNB_STACK_GROUP_PATH` | `./stack-group.toml`    | Path to group definition(#buildpacks-directory-layout)) of buildpacks that run during the extend phase that created the image being rebased.
+
+#### `extender`
+Usage:
+```
+/cnb/lifecycle/extender \
+  [-stack-buildpacks <stack-buildpacks>] \
+  [-stack-group <stack-group>] \
+  [-layers <layers>] \
+  [-log-level <log-level>] \
+  [-plan <plan>] \
+  [-stack-platform <stack-platform>]
+```
+
+##### Inputs
+| Input          | Env                   | Default Value     | Description
+|----------------|-----------------------|-------------------|----------------------
+| `<layers>`     | `CNB_LAYERS_DIR`      | `/layers`         | Path to layers directory
+| `<log-level>`  | `CNB_LOG_LEVEL`       | `info`            | Log Level
+| `<plan>`       | `CNB_PLAN_PATH`       | `./plan.toml`     | Path to resolved build plan (see [`plan.toml`](#plantoml-toml))
+| `<stack-platform>`   | `CNB_STACK_PLATFORM_DIR`    | `/stack-platform`       | Path to stack-specific platform directory
+| `<stack-buildpacks>`  | `CNB_STACK_BUILDPACKS_DIR`  | `/cnb/stack/buildpacks` | Path to stack buildpacks directory (see [Buildpacks Directory Layout]
+| `<stack-group>`       | `CNB_STACK_GROUP_PATH`      | `./stack-group.toml`    | Path to output group definition(#buildpacks-directory-layout))
+
+##### Outputs
+| Output                                     | Description
+|--------------------------------------------|----------------------------------------------
+| [exit status]                              | (see Exit Code table below for values)
+| `/dev/stdout`                              | Logs (info)
+| `/dev/stderr`                              | Logs (warnings, errors)
+| `<layers>/<buildpack ID>.tgz`              | Layer snapshot (see [Buildpack Interface Specfication](buildpack.md)
+| `<layers>/stack-layer.toml`                | Layer snaphot metadata
+| `<layers>/config/metadata.toml`            | Build metadata (see [`metadata.toml`](#metadatatoml-toml))
```

Comment:
Same comment as above. Where do the files from `[[excludes]]` end up?

Reply:
Fixed in 146d0d5